cmd/piper_test.go

```diff
@@ -63,7 +64,7 @@ func (m *shellMockRunner) RunShell(s string, c string) error {
 	if m.shouldFailWith != nil {
 		return m.shouldFailWith
 	}
-
+	m.shell = append(m.shell, s)
```

Comment:
See also above. Why having the result recorded twice?

Reply:
see answer above ...